md/bdm-management-in-bonita-bpm-portal.md

```diff
@@ -13,11 +13,12 @@ The Business objects must match the structure used by the deployed processes. Ma
 
 ::: info
 **:fa-info-circle: Note:** The deployment of a Business Data Model requires pausing the service during the operation, so that the update can be performed without affecting ongoing processes. 
+ For Efficiency and Performance editions, you must not have [access control](#installAccessControl) file installed to install or update Business Data Model.
```

Comment:
must have no access control installed in order to be able to install of update a Business Data Model